vyper/builtins/functions.py

```diff
@@ -1061,6 +1061,26 @@ def fetch_call_return(self, node):
 
         kwargz = {i.arg: i.value for i in node.keywords}
 
+        delegate_call = kwargz.get("is_delegate_call", False)
+        static_call = kwargz.get("is_static_call", False)
+        if delegate_call and static_call:
+            raise ArgumentException(
+                "Call may use one of `is_delegate_call` or `is_static_call`, not both", node
+            )
+
+        value = kwargz.get("value")
+        if (delegate_call or static_call) and value is not None:
```

Comment:
this check is not going to do the right thing on raw_calls like `raw_call(..., is_delegate_call=False, value=123)`